src/components/Confirmation.lua

```diff
@@ -111,6 +111,8 @@ function component:render()
 					e.TButton {
 						Text = 'Okay';
 						Size = UDim2.new(0, 100, 0, 27);
+                        TextColor3 = Color3.new(0, 0, 0);
+                        BackgroundColor3 = Color3.new(1, 0.266667, 0.266667);

```

Comment:
Looks like there is a difference in indentation here again.

This is a very good design pattern! We might break this out into a reusable component in the future.